application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -123,24 +124,29 @@ private void handleAddCommand(SlashCommandInteractionEvent event) {
 
     private void handleListCommand(SlashCommandInteractionEvent event) {
         BiFunction<Long, Instant, String> getDescription = (channelId, remindAt) -> """
-                Channel: <#%s>
-                Remind at: <t:%s>""".formatted(channelId, remindAt.getEpochSecond());
-
-        EmbedBuilder remindersEmbed =
-                new EmbedBuilder().setTitle("Reminders").setColor(RemindRoutine.AMBIENT_COLOR);
-
-        database.read(context -> {
-            try (SelectWhereStep<PendingRemindersRecord> select =
-                    context.selectFrom(PENDING_REMINDERS)) {
-                return select
-                    .where(PENDING_REMINDERS.GUILD_ID.eq(event.getGuild().getIdLong())
-                        .and(PENDING_REMINDERS.AUTHOR_ID.eq(event.getUser().getIdLong())))
-                    .fetch();
-            }
-        })
+                Channel: %s
+                Remind at: %s""".formatted(MessageUtils.mentionChannelById(channelId),
+                TimeFormat.DEFAULT.format(remindAt));
+
+        EmbedBuilder remindersEmbed = new EmbedBuilder().setTitle("Pending reminders")
+            .setColor(RemindRoutine.AMBIENT_COLOR);
+
+        long guildId = event.getGuild().getIdLong();
+        long userId = event.getUser().getIdLong();
+
+        database
+            .read(context -> context.selectFrom(PENDING_REMINDERS)
+                .where(PENDING_REMINDERS.GUILD_ID.eq(guildId)
+                    .and(PENDING_REMINDERS.AUTHOR_ID.eq(userId)))
+                .orderBy(PENDING_REMINDERS.CREATED_AT.asc())
+                .fetch())
```

Comment:
id move this into a help method to increase readability of this method and reduce its scope